mmv1/products/cloudrunv2/Service.yaml

```diff
@@ -772,6 +772,26 @@ properties:
                 Containers which should be started before this container. If specified the container will wait to start until all containers with the listed names are healthy.
               item_type:
                 type: String
+            - name: 'baseImageUri'
+              type: String
+              description: |-
+                Base image for this container. Only supported for services. If set, it indicates that the service is enrolled into automatic base image update.
```

Comment:
"Only supported for services" can be removed.

Reply:
It hasn't been tested for jobs, so I'd rather keep it.

Reply:
But this file is for service only. This field is not shared with other resources

Reply:
Removed.